opentelekomcloud/services/elb/v3/resource_opentelekomcloud_lb_member_v3.go

```diff
@@ -0,0 +1,198 @@
+				Optional:     true,
+				ValidateFunc: validation.StringLenBetween(0, 255),
+			},
+			"project_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"protocol_port": {
+				Type:         schema.TypeInt,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.IsPortNumber,
+			},
+			"subnet_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+			"weight": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Computed:     true,
+				ValidateFunc: validation.IntBetween(0, 100),
+			},
+			"member_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"operating_status": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"ip_version": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceLBMemberV3Create(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	config := meta.(*cfg.Config)
+	client, err := common.ClientFromCtx(ctx, keyClient, func() (*golangsdk.ServiceClient, error) {
+		return config.ElbV3Client(config.GetRegion(d))
+	})
+	if err != nil {
+		return fmterr.Errorf(ErrCreateClient, err)
+	}
+
+	opts := members.CreateOpts{
+		Address:      d.Get("address").(string),
+		ProtocolPort: d.Get("protocol_port").(int),
+		Name:         d.Get("name").(string),
+		ProjectID:    d.Get("project_id").(string),
+		SubnetID:     d.Get("subnet_id").(string),
+	}
+	if w, ok := d.GetOk("weight"); ok {
+		weight := w.(int)
+		opts.Weight = &weight
+	}
+	member, err := members.Create(client, poolID(d), opts).Extract()
+	if err != nil {
+		return fmterr.Errorf("error creating LB pool member v3: %w", err)
+	}
+	_ = d.Set("member_id", member.ID) // this can't ever return an error
+
+	if err := common.SetComplexID(d, "pool_id", "member_id"); err != nil {
+		return diag.FromErr(err)
+	}
+
+	clientCtx := common.CtxWithClient(ctx, client, keyClient)
+	return resourceLBMemberV3Read(clientCtx, d, meta)
+}
+
+func resourceLBMemberV3Read(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	config := meta.(*cfg.Config)
+	client, err := common.ClientFromCtx(ctx, keyClient, func() (*golangsdk.ServiceClient, error) {
+		return config.ElbV3Client(config.GetRegion(d))
+	})
+	if err != nil {
+		return fmterr.Errorf(ErrCreateClient, err)
+	}
+
+	member, err := members.Get(client, poolID(d), memberID(d)).Extract()
+	if err != nil {
+		return diag.FromErr(common.CheckDeleted(d, err, "error reading LB pool member v3"))
+	}
+	mErr := multierror.Append(
+		d.Set("address", member.Address),
+		d.Set("name", member.Name),
+		d.Set("operating_status", member.OperatingStatus),
+		d.Set("project_id", member.ProjectID),
+		d.Set("subnet_id", member.SubnetID),
+		d.Set("protocol_port", member.ProtocolPort),
+		d.Set("weight", member.Weight),
+		d.Set("ip_version", member.IpVersion),
```

Comment:
can you set `member_id`?

Reply:
`member_id` is part of the resource ID, it's used to make a `Get` request, and I suppose it shouldn't be set during the read operation.